### DISCUSSION TITLE
Availability of the NDI5 Alpha

### DISCUSSION THREAD

**Discussion by APz** [DistroAV/DistroAV#807] [General]
The Alpha was downloadable some time ago and I have it installed, but when starting with a fresh OBS installation, I noticed all the downloadable files from the Alpha build have expired. Are they still somewhere to download, or alternatively would someone still have the x86 Windows build installer package in around?

**Comment by Trouffman** [comment]
NDI 5 RUntim are available at : http://new.tk/NDIRedistV5
